com.unity.ml-agents/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 ### Major Changes
 
 ### Minor Changes
+Model checkpoints are now also saved as .nn files during training. (#4127)
```

Comment:
minor, but in the past I believe we used a bulleted list
```suggestion
* Model checkpoints are now also saved as .nn files during training. (#4127)
```

Reply:
Yup, you're right. Looks like we're using `-` for bullets though.